tests/Mutations-E2E/Scalar/04_Mutation_Scalar_CallArgument_Test.cpp

```diff
@@ -0,0 +1,61 @@
+#include "FixturePaths.h"
+
+#include "mull/Bitcode.h"
+#include "mull/MutationPoint.h"
+#include <mull/Mutators/ScalarValueMutator.h>
+
+#include <clang/Frontend/ASTUnit.h>
+#include <clang/Tooling/Tooling.h>
+#include <llvm/IR/Module.h>
+
+#include "Helpers/MutationTestBed.h"
+
+#include <gtest/gtest.h>
+
+using namespace mull;
+using namespace mull_test;
+using namespace llvm;
+
+static FilePathFilter nullPathFilter;
+
+static const std::string testCode = std::string(R"(///
+int dummy(int a) {
+  return a;
+}
+int caller() {
+  return dummy(0);
```

Comment:
What is the expected behavior in case there are two arguments, e.g. `dummy(0, 5)`? I'm afraid this is not supported by Mull in the current implementation.